src/docfx/build/page/BuildPage.cs

```diff
@@ -178,14 +178,25 @@ private static (List<Error> errors, Schema schema, OutputModel model)
             LoadSchemaDocument(Context context, List<Error> errors, JToken token, Document file, Action<Document> buildChild)
         {
             var obj = token as JObject;
-            if (file.Schema is null)
+            if (file.Schema is null || !File.Exists($"data/{file.Schema.Type.Name}.json"))
             {
                 throw Errors.SchemaNotFound(file.Mime).ToException();
             }
 
-            // todo: why not directly use strong model here?
-            var (schemaViolationErrors, content) = JsonUtility.ToObject(token, file.Schema.Type, transform: AttributeTransformer.TransformSDP(context, file, buildChild));
-            errors.AddRange(schemaViolationErrors);
+            // TODO: load json schema from template
+            var schemaFilePath = $"data/{file.Schema.Type.Name}.json";
+            var jsonSchema = JsonUtility.Deserialize<JsonSchema>(File.ReadAllText(schemaFilePath), schemaFilePath);
```

Comment:
Load schema only once, instead of per file.

Reply:
good suggestion